functions/defaulted_parameters.md

```diff
@@ -0,0 +1,87 @@
+# Module name: topic name
+_Skeleton instructions are typeset in italic text._
+
+## Overview
+
+Functions in C++ may be overloaded with different numbers and types of parameters. It may be of value to specify default values for some number of parameters, to allow a consumer to avoid specifying parameters that rarely change, or to enable expanding the set of parameters while maintaining backward compatibility with existing consumers.
```

Comment:
Nit: Could you break this long line into 75- or 100-character lines?

`consumer` should presumably be `caller` (twice).

Reply:
I can certainly do either. Do we have a preferred convention?

Reply:
I updated free sections to 75 characters. Havent yet mucked around with list items. Still learning some of these features of markdown...